scrape_repo_information.py

```diff
@@ -0,0 +1,88 @@
+import urllib.request
+import pdb
+import xml.etree.ElementTree as xml
+import argparse, urllib.request
+import json
+import collections
+import time
+import urllib
+import os
+"""
+Script to scrape text from GitHub and into JSON format for all language pairs in Apertium.
+Stem counter by sushain
+"""
+if "GITHUB_CLIENT_ID" in os.environ:
+    CLIENT_ID = os.environ["GITHUB_CLIENT_ID"]
+else:
+    CLIENT_ID = ""
+    print("There doesn't seem to be a GitHub Client ID; the scraping process may not be fast")
+if "GITHUB_SECRET_CLIENT_ID" in os.environ:
+    CLIENT_SECRET = os.environ["GITHUB_SECRET_CLIENT_ID"]
+else:
+    CLIENT_SECRET = ""
+    print("There doesn't seem to be a GitHub Client Secret; the scraping process may not be fast")
```

Comment:
As I mentioned in my last comment, please include a link tot he GitHub docs on how to get one.